lib/documents.js

```diff
@@ -37,7 +37,11 @@ function Paragraph(children, properties) {
         styleId: properties.styleId || null,
         styleName: properties.styleName || null,
         numbering: properties.numbering || null,
-        alignment: properties.alignment || null
+        alignment: properties.alignment || null,
+        indentLeft: properties.indentLeft || null,
+        indentRight: properties.indentRight || null,
+        indentFirstLine: properties.indentFirstLine || null,
+        indentHanging: properties.indentHanging || null
```

Comment:
I wonder whether these should be grouped together into an `indent` object? It seems nicer to keep related properties together, although having them flat is possibly simpler. Thoughts?